lib/rubocop/cop/team.rb

```diff
@@ -174,6 +174,12 @@ def support_target_ruby_version?(cop)
       end
 
       def support_target_rails_version?(cop)
+        if cop.class.respond_to?(:gem_requirements)
```

Comment:
It looks like the versioning between `rubocop` and `rubocop-rails` gets a bit tricky here.

I'd love to delete this method entirely, but I think it still needs to support calling `support_target_rails_version?` on older versions that don't yet have the `requires_gem` and `gem_requirements` API